vtr_flow/scripts/python_libs/vtr/task.py

```diff
@@ -497,12 +497,24 @@ def create_job(
     Create an individual job with the specified parameters
     """
     param_string = "common" + (("_" + param.replace(" ", "_")) if param else "")
-    for spec_char in [":", "<", ">", "|", "*", "?"]:
+    
+    #remove any address-related characters that might be in the param_string
```

Comment:
Could say what these address characters are and how you're replacing them @saaramahmoudi